views/director-residential-address.html

```diff
@@ -18,9 +18,51 @@
       
       <form action="" method="post">
 
+        <span class="govuk-caption-xl govuk-!-padding-bottom-1">Doe, John Doe</span>
```

Comment:
Director name is hard coded, should be passed in from filing I believe?

Reply:
Yes all the values are hardcoded as it is dependent on the previous page (which is yet to be done). So figured all the values will be populated once that is ready

Reply:
Director name page is already implemented so we can pull name from filing. See occupation page html/controller for an example

Reply:
Name added